apps/api/infra/api.ts

```diff
@@ -310,6 +320,11 @@ export const serviceSetup = (services: {
       max: 50,
       min: 10,
     })
+    .replicaCount({
+      default: 10,
+      max: 50,
+      min: 10,
+    })
```

Comment:
This looks very high. Are you sure you need at min 10 pods and won't allow it to scale down to something sensible like 2 pods under no load such as at night?

Reply:
This is also a duplicate of the config above it so it should at least be removed before merge

Reply:
I believe this was changed when there was some issue with deploying the feature at some point, and was left in by accident. I'll remove the duplicates.

Reply:
Regarding the 10 pods though,
On main today this is set up like this:

```
    .replicaCount({
      default: 10,
      max: 50,
      min: 10,
    })
```

I guess that's a discussion for some other PR.